### PR TITLE
Types 2.0 - Fix JS-quantities

### PR DESCRIPTION
**Pull request by rummelsworth**
Please fill in this template.

- [X] Prefer to make your PR against the `types-2.0` branch.
- [X] Test the change in your own code.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [X] Run `npm run lint -- package-name` if a `tslint.json` is present.

If changing an existing definition:
- [X] Provide a URL to  documentation or source code which provides context for the suggested changes: `http://gentooboontoo.github.io/js-quantities/` _Previous declaration file differed significantly from documented API. Previous test file covered only one API point out of several dozen actual._

### CODE REVIEW

**Comment by wrummler** [comment]
This PR very likely needs more work. Though the documented DefinitelyTyped test procedure passes for these changes, the TS I've written for a separate project that depends on js-quantities no longer compiles. I've since then tried and failed to adapt the TS Handbook's declaration file templates, but no dice so far.

Closing this until I know what's going on/what the hell I'm doing. :8ball:

**Comment by wrummler** [comment]
So I think I fixed the problems I was encountering. DT tests pass, lint is clean. Usage in my own project works without any issues. Reopening.

File: js-quantities/index.d.ts [hunk 1]
```diff
@@ -1,79 +1,73 @@
-// Type definitions for JS-quantities
+// Type definitions for JS-quantities 1.6
 // Project: http://gentooboontoo.github.io/js-quantities/
-// Definitions by: William Comartin <https://github.com/wcomartin>
+// Definitions by: William Rummler <https://github.com/wrummler>
```
Comment (vvakame) [inline js-quantities/index.d.ts]: sorry for late respobse.
please keep original author name & append you name below.

Comment (wrummler) [inline js-quantities/index.d.ts]: @vvakame No need to apologize, I have no claim to your time. Many thanks to you and all the other maintainers of this project.

I'm not really comfortable making the requested revision, at least on the basis of whether there was any level of meaningful contribution in the original definitions. The original definitions were more than a little mismatched with this library's actual API, as can be verified by testing the original version of this file against the test file submitted in this PR. The effect of the originals' inaccuracy wasn't neutral, either. My team (new to TS and the modern JS ecosystem in general) lost some time at one point until realizing our problems were due to the non-trivial inaccuracies of these definitions.

If DT policy requires this revision anyway, please let me know and I'll make the necessary update. Thanks again.

Comment (wcomartin) [inline js-quantities/index.d.ts]: I have no problem with @wrummler removing my name as my contribution was not complete and was only to get the definition working in my project.

Comment (andy-ms) [inline js-quantities/index.d.ts]: OK then. (Thanks for linting too!)

**Pull request merged**